Jenkinsfile

```diff
@@ -135,7 +135,8 @@ def CloudInstallAndTest(cloudTarget) {
     sh """
     ls -lh *.whl
     pip3 install *.whl
-    pip3 install tensorflow
+    sudo pip3 install tensorflow
```

Comment:
need sudo here because it installs `toco_from_protos` to `/usr/local/bin/`

Reply:
Trying to understand, why `toco_from_protos ` has to be installed under `/usr/local/bin`? Can we use `pip3 install --user tensorflow`?

Reply:
Since it is in Jenkinsfile and used only by CI we agreed to keep sudo